### PR TITLE
Update PassiveData sample for Health Services beta01

### PR DESCRIPTION
**Pull request by ithinkihaveacat**

### CODE REVIEW

File: health-services/PassiveData/app/build.gradle [hunk 1]
```diff
@@ -61,7 +65,7 @@ dependencies {
     // Wear
     implementation 'androidx.wear:wear:1.2.0'
     // Health Services
-    implementation "androidx.health:health-services-client:1.0.0-alpha02"
+    implementation 'androidx.health:health-services-client:1.0.0-SNAPSHOT'
```
Comment (garanj) [inline health-services/PassiveData/app/build.gradle:68]: This will be beta01 right?

**Review by garanj — APPROVED** [review]

File: health-services/PassiveData/app/build.gradle (hunk 1 above)
Comment (ithinkihaveacat) [reply health-services/PassiveData/app/build.gradle:68]: Yes! Keeping as the SNAPSHOT for now so the thing builds.

File: health-services/PassiveData/app/src/main/java/com/example/passivedata/HealthServicesManager.kt [hunk 2]
```diff
@@ -16,44 +16,44 @@
...
+            PassiveDataService::class.java,
+            passiveListenerConfig
+        ).await()
     }
 
     suspend fun unregisterForHeartRateData() {
-        Log.i(TAG, "Unregistering for background data.")
-        passiveMonitoringClient.unregisterDataCallback().await()
+        Log.i(TAG, "Unregistering listeners")
+        passiveMonitoringClient.clearPassiveListenerCallbackAsync().await()
```
[the remark this answers is not in the document]
Comment (ithinkihaveacat) [reply health-services/PassiveData/app/src/main/java/com/example/passivedata/HealthServicesManager.kt]: No, I guess that's unnecessary, removed in 9011ffc.

File: health-services/PassiveData/app/src/main/java/com/example/passivedata/PassiveDataRepository.kt [hunk 3]
```diff
@@ -16,11 +16,18 @@
 
 package com.example.passivedata
 
+import android.util.Log
```
[the remark this answers is not in the document]
Comment (ithinkihaveacat) [reply health-services/PassiveData/app/src/main/java/com/example/passivedata/PassiveDataRepository.kt]: No, removed.

**Pull request merged**